vcd/datasource_vcd_vapp_vm.go

```diff
@@ -311,6 +320,12 @@ func datasourceVcdVAppVm() *schema.Resource {
 				Computed:    true,
 				Description: "True if the virtual machine supports addition of memory while powered on.",
 			},
+			"sizing_policy_id": {
```

Comment:
SDK upgrade adds new fields to data source?

Reply:
No. This was a leftover. In its original implementation, this field was forgotten, and the enhanced error detection of SDK 2.x created a panic because this one was missing. 
I have added it to make the data source complete.